resources/scholarships.md

```diff
@@ -0,0 +1,5 @@
+# Supportive Tech Organizations
+
+Discover valuable resources to support your educational journey and access financial aid for online courses:
+
+- **[Coursera Financial Aid Guide](https://github.com/ZeeshanMukhtar1/Coursera-Financial-Aid-Application)**: A comprehensive guide on securing financial aid for Coursera courses. Created and maintained by [Zeeshan Mukhtar](https://www.linkedin.com/in/zeeshanmukhtar1/).
```

Comment:
```suggestion
- **[Coursera Financial Aid Guide](https://github.com/ZeeshanMukhtar1/Coursera-Financial-Aid-Application)**: A comprehensive guide on securing financial aid for Coursera courses.
```

We don't need to mention who created and maintained here. Folks will know right away once they visit the link. 🙂